### DISCUSSION TITLE
NNP training on structures with large vacuum space and gas bubbles

### DISCUSSION THREAD

**Discussion by lnnbig** [deepmodeling/deepmd-kit#2368] [Q&A]
Hello everyone,

I train NNPs on H2O + CO2  mixtures (from AIMD at 0.1 bar, 300-1000 K). At these pressure-temperature conditions, the structures contain two phases: a liquid and a coexisting gas phase, and these two phases have different H2O/CO2 ratios (because CO2 is more volatile compared to H2O). Thus these structures are very heterogeneous. I get very large RMSE energy >100 meV/atom and RMSE force >500 meV/Å on the training set.

I was wondering if this large error is related to the structural complexity and heterogeneity of my training set. 
Does increasing the embedding/fitting-net size (currently they are 25,50,100 and 240,240,240) help reduce the RMSE values? 

Many thanks for your help.

**Comment by jameswind** [comment]
Could you have a look at the std of forces, and maybe show your network
architecture and training parameters (in your train.json)?
Also, what’s the average number of atoms in each of your configuration?
Then how about the AIMD simulation? What’s the software you used and how
did you convert the data into training data?
